rpm/xcache.spec

```
@@ -202,7 +215,16 @@ mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/90-cms-xcache-local-redirector.cfg
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/95-cms-xcache-logging.cfg
 
+
+%files -n xcache-redir
+%config %{_sysconfdir}/xrootd/xrootd-xcache-redir.cfg
+%config %{_sysconfdir}/xrootd/config.d/03-redir-tunning.cfg
```

```suggestion
%config %{_sysconfdir}/xrootd/config.d/03-redir-tuning.cfg
```
Also rename the file